docs/validation-rules.rst

```diff
@@ -7,6 +7,8 @@ This can be used in conjunction with the  `schema <schema_dict-rule>`_ rule
 when validating a mapping in order to set the
 :attr:`~cerberus.Validator.allow_unknown` property of the validator for the
 subdocument.
+This rule has precedence over ``purge_unknown``
+(see :ref:`purging-unknown-fields`).
 For a full alaboration refer to :ref:`this paragraph <allowing-the-unknown>`.
```

Comment:
you **might** also fix this unrelated typo (**e**laboration). afaict @nicolaiarocci's merges usually around sunrise in europe.

Reply:
Should I do it in another commit or in same? (asking because different projects have different rules for these kind of fixes)

Reply:
there's no rule regarding this around here, at least not for such minor stuff. documentation improvements are very appreciated.

Reply:
Ok, fixed.

Reply:
mille gracie.